src/rector.php

```diff
@@ -19,17 +19,20 @@
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->skip([
-        __DIR__ . '/vendor',
-        __DIR__ . '/rezolve_rce_coding_standard',
-        __DIR__ . '/Tests',
+        '*/rezolve_rce_coding_standard',
+        '*/Tests',
+        '*/Test',
+        '*/Fixtures',
         /**
          * remove rules
          */
         Rector\CodeQuality\Rector\If_\ExplicitBoolCompareRector::class,
         Rector\Php54\Rector\Array_\LongArrayToShortArrayRector::class,
         Rector\CodingStyle\Rector\ClassConst\VarConstantCommentRector::class,
         Rector\CodingStyle\Rector\Stmt\NewlineAfterStatementRector::class,
-
+        Rector\CodingStyle\Rector\Catch_\CatchExceptionNameMatchingTypeRector::class,
+        Rector\CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector::class,
+        Rector\CodingStyle\Rector\Encapsed\WrapEncapsedVariableInCurlyBracesRector::class,
```

Comment:
Disable rules:

**CatchExceptionNameMatchingTypeRector** - Rector requires to name exceptions: `} catch (NotFound404Exception $notFound404Exception) {`

**EncapsedStringsToSprintfRector** - to replace double-quoted strings with `sprintf`: 
![image](https://user-images.githubusercontent.com/127951620/236449852-10d93b01-753f-4154-9ee5-f4a389442c02.png)

**WrapEncapsedVariableInCurlyBracesRector** - require to use curly braces `"string {$var}"`. Removed in PHP8